geode-dunit/src/main/java/org/apache/geode/test/dunit/rules/CacheRule.java

```diff
@@ -242,6 +242,7 @@ private void closeCache() {
     }
   }
 
+
```

Comment:
Your description mentions modifying CacheRule to wait for close to complete, but the only change to the file is the addition of a blank line. Is something missing?

Reply:
Sorry, I had originally modified CacheRule and added a closeAndWaitForCallback where it would register a CacheLifecycleListener, call close and wait for the listener to receive a close event.  I ended up moving the code to a Membership listener into the test itself.

Reply:
I'll remove this newline